Autopilot/AttitudeManager/OutputMixing.cpp

```diff
@@ -25,17 +32,16 @@ static void constrainOutput(float *channelOut);
 OutputMixing_error_t OutputMixing_Execute(PID_Output_t *PidOutput, float *channelOut)
 {
 
-#if TAIL_TYPE != INV_V_TAIL
-	#error "Requested tailtype not supported !"
-#endif
-
 	OutputMixing_error_t error;
 	error.errorCode = checkInputValidity(PidOutput);
 
-    PidOutput->yawPercent += PidOutput->rollPercent * ADVERSE_YAW_MIX; // mix roll rate into rudder to counter adverse yaw
-
+#if TAIL_TYPE == INV_V_TAIL
     channelOut[L_TAIL_OUT_CHANNEL] =  (PidOutput->yawPercent * RUDDER_PROPORTION) - (PidOutput->pitchPercent * ELEVATOR_PROPORTION); //Tail Output Left
```

Comment:
in `Autopilot/Test/Src/AttitudeManager/Test_OutputMixing.cpp` you only test if `ELEVATOR_OUT_CHANNEL` and `RUDDER_OUT_CHANNEL` are within [-100f, 100f]. (Line 262)

I may be missing something, but if you are using `L_TAIL_OUT_CHANNEL` and `R_TAIL_OUT_CHANNEL` when `TAIL_TYPE == INV_V_TAIL`, would we want to also test to see if those values are within [-100f, 100f] as well?

Reply:
ya, it's a point. I wasn't too sure what to do about that. Cuz testing both setups would mean having to build all the unit tests multiple times (one for each aircraft). I figured we could just unit test the default setup (which is Infinion). If some day someone wants to put in the infrastructure to cover all the setups, that'd be fine, but truthfully, I don't think it's valuable enough to set it up now. (let me know if you disagree)